regulations/views/comment.py

```diff
@@ -70,3 +71,29 @@ def submit_comment(request):
         'status': 'submitted',
         'metadata_url': metadata_url,
     })
+
+
+@csrf_exempt
+@require_http_methods(['GET', 'HEAD'])
+def get_federal_agencies(request):
+    url = "{}?{}".format(settings.REGS_GOV_API_LOOKUP_URL,
+                         "field=gov_agency&dependentOnValue=Federal")
+    response = requests.get(
+        url,
+        headers={'X-Api-Key': settings.REGS_GOV_API_KEY}
+    )
+    response.raise_for_status()
```

Comment:
Yes, it would make sense to _handle_ errors better. I am also planning to add caching, timeouts and retries.